spec/lib/fixtures/state-multi.yml

```diff
@@ -51,6 +51,17 @@ pod:
     podIP: 1.2.3.4
     containerStatuses:
     - imageID: docker://ccc
+- metadata:
+    uid: 650aa4a8-2034-55c1-gge9-2451d6a2799b
+    name: pending-pod-0
+    namespace: namespace
+    labels:
+      skiff-role-name: pending
+    annotations:
+      skiff-exported-properties-dummy: '{}'
+  status:
+    podIP: 1.2.3.4
+    containerStatuses:
```

Comment:
It would probably better to make it explicitly nil (`containerStatuses: ~`) to make sure we don't accidentally edit it out.

… I'll just make it a separate commit on top since it's so trivial; hopefully you don't mind.

Reply:
great, thanks!